docs/source_values.rst

```diff
@@ -178,12 +185,106 @@ In this example, the LED is lit only when the button is not pressed:
 
 Some tools combine the values of multiple sources:
 
-.. image:: images/combining_sources.*
+.. image:: images/source_values/combining_sources.*
     :align: center
 
 In this example, the LED is lit only if both buttons are pressed (like an
 `AND`_ gate):
 
+.. image:: images/source_values/combining_sources_led_buttons.*
+    :align: center
+
 .. literalinclude:: examples/combining_sources.py
 
 .. _AND: https://en.wikipedia.org/wiki/AND_gate
+
+While most devices have a :attr:`~Device.value` range between 0 and 1, some have
+a range between -1 and 1 (e.g. :class:`Motor`, :class:`Servo` and
+:class:`TonalBuzzer`). Some source tools output values between -1 and 1, which
+are ideal for these devices, for example passing :func:`~tools.sin_values` in:
+
+.. image:: images/source_values/sin_values.*
+    :align: center
+
+.. literalinclude:: examples/sin_values.py
+
+In this example, all three devices are following the `sine wave`_. The motor
+ramps up from 0 (stopped) to 1 (full speed forwards), then back down to 0 and on
+to -1 (full speed backwards) in a cycle. Similarly, the servo moves from its mid
+point to the right, then towards the left; and the buzzer starts with its mid
+tone, gradually raises its frequency, to its highest tone, then down towards its
+lowest tone. Note that setting :attr:`~SourceMixin.source_delay` will alter
+the speed at which the device iterates through the values.
```

Comment:
Might be worth including an example here, so that people can see how / where the `source_delay` is set? I guess a nice example might be setting `source_delay` to three different values (and ignoring my previous comment about setting `servo.source = motor` :wink: ) so that the three different devices ramp at three different speeds?

Reply:
The candle example sets source_delay, that'll do for now.

Reply:
Ahh, fair enough. But if that's the only place where source_delay is demonstrated, it might be worth pointing it out? (e.g. "Note that this example also uses `source_delay` to slow down the speed of the flickering")

Should https://gpiozero.readthedocs.io/en/master/source_values.html#source-tools and https://gpiozero.readthedocs.io/en/master/api_tools.html#gpiozero.tools.random_values be using the same code snippet?

Reply:
Added a note